packages/client/src/components/DishPopup/DaysList.tsx

```diff
@@ -0,0 +1,41 @@
+import Box from '@mui/material/Box';
+import FormControl from '@mui/material/FormControl';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
+import * as React from 'react';
+
+const DaysList = () => {
+  const [mealCount, setMealCount] = React.useState('');
```

Comment:
Chyba sensowniej żeby ilość dań była liczbą?

Reply:
nie wiem czy tak da się zrobić, bo niżej jest ten event.target.value i to nie moze number przyjmować

Reply:
`parseInt(event.target.value)`?